modules/recaptcha/modules.php

```diff
@@ -54,9 +54,8 @@ protected function output() {
  */
 class Hm_Output_recaptcha_form extends Hm_Output_Module {
     protected function output() {
-        if (!$this->get('router_login_state')) {
-            $rconf = $this->get('recaptcha_config');
-            return '<div class="g-recaptcha" data-sitekey="'.$this->html_safe($rconf['site_key']).'"></div>';
+        if (!$this->get('router_login_state') && $this->get('recaptcha_config')) {
```

Comment:
use the condition like it's used here: https://github.com/cypht-org/cypht/blob/master/modules/recaptcha/modules.php#L16C9-L17C55

you can put the same condition at this level too: Hm_Output_recaptcha_script(https://github.com/cypht-org/cypht/blob/master/modules/recaptcha/modules.php#L46)